tutorials/model_command.md

```diff
@@ -0,0 +1,271 @@
+\page model_command Model Command
+
+## Overview
+This command allows you to get the following information about models in the running world:
```

Comment:
What about changing it a bit:
```
`ign model` command allows you to get information about the models for a given running `ignition` simulation.

For each model, it is possible to get information about its
 -  pose: Pose of the model
 -  links: Pose, mass, and inertial matrix of the link
 -  joints: Parent link, child link and joint type.

```

I would remove the list below given that it is too verbose.

Reply:
Addressed